packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -3119,14 +3141,19 @@ describe('KeyringController', () => {
         it('should throw error when using the wrong password', async () => {
           await withController(
             {
-              skipVaultCreation: true,
               cacheEncryptionKey,
-              state: { vault: 'my vault' },
+              skipVaultCreation: true,
+              state: {
+                vault: freshVault,
+                // @ts-expect-error we want to force the controller to have an
```

Comment:
Nit: Hmm. I don't quite understand why we would need this here, as the `encryptionSalt` state is _not used_ for anything other than detecting a mismatch between `encryptionKey` and `vault` (which is not relevant for this test, certainly because `encryptionKey` is not even present).

I see that it is a pre-existing widespread pattern in these tests though, so we can fix it later.